packages/gatsby/index.d.ts

```diff
@@ -795,7 +795,7 @@ export interface GatsbySSR<
 }
 
 export interface PluginOptions {
-  plugins: unknown[]
+  // plugins: unknown[]
```

Comment:
This did throw type errors for me.

Alternative from removing it, would be to make plugins optional?

Reply:
Always? How? Why?

This mdx-plugin has no need for this option!

Reply:
Its when the plugin got processed by Gatsby. So if you do
```
exports.onPreInit = (args, pluginOptions) => {
 // pluginoptions always has plugins
}
```

Reply:
Removed :)